### PR TITLE
Make instrument tiles clickable and add tooltips

### PR DESCRIPTION
**Pull request by jadalilleboe**
@mcoughlin Here's the interaction:
ezgif com-gif-maker (5)
could be annoying to click a bunch of those tiles though...
the button handler just logs the selected instrument field ids as of now

### CODE REVIEW

**Comment by mcoughlin** [comment]
@jadalilleboe Looks amazing. Does it also work for the observation plan tiles after making an observation plan at the bottom of the page?

**Comment by jadalilleboe** [comment]
No, the selected state is in GcnSelectionForm right now. I think the state would have to be lifted up to be sent along on submit of the observation plan form

**Comment by mcoughlin** [comment]
But can replicate that same button in the observationplanform or whatever we call it right?

**Comment by jadalilleboe** [comment]
@mcoughlin yes most likely. So the observation plan tiles are different than the instrument tiles? How do I get those to show up? I think I could replicate the same logic for those

**Comment by jadalilleboe** [comment]
ohhh ok. So how do I get the tiles to show up on these ones? they just have the black lines for me
Screen Shot 2022-03-31 at 10 58 53 AM

**Comment by mcoughlin** [comment]
@jadalilleboe super cool. Let's get this merged like this and will deal with using them next.

**Comment by mcoughlin** [comment]
@jadalilleboe when you have a second, could you return the demo file to normal and remove console statements and such? Would like to get this reviewed and in to show off in the demo on Tuesday.

File: static/js/components/GcnSelectionForm.jsx [hunk 1]
```diff
@@ -190,6 +190,11 @@ const GcnSelectionForm = ({ gcnEvent }) => {
     setIsSubmitting(false);
   };
 
+  // const handleSendToObservationPlan = (instrument) => {
```
Comment (mcoughlin) [inline static/js/components/GcnSelectionForm.jsx:193]: Can you bring this back?

File: static/js/components/GcnSelectionForm.jsx [hunk 2]
```diff
@@ -304,6 +312,14 @@ const GcnSelectionForm = ({ gcnEvent }) => {
         />
       </div>
       <div>
+        <Button
+          variant="outlined"
+          // onClick={() =>
```
Comment (mcoughlin) [inline static/js/components/GcnSelectionForm.jsx:317]: here too.

File: static/js/components/ObservationPlanRequestLists.jsx [hunk 3]
```diff
@@ -109,6 +109,11 @@ const ObservationPlanGlobe = ({ observationplanRequest, loc }) => {
       setObsList(response.data);
     };
     fetchObsList();
+    if (obsList) {
```
Comment (mcoughlin) [inline static/js/components/ObservationPlanRequestLists.jsx:112]: This just sets the defaults I guess?

**Review by mcoughlin — APPROVED** [review]
Looking overall very good. Looks like a few things commented out that shouldn't be, otherwise looks good to go.

File: static/js/components/GcnSelectionForm.jsx (hunk 1 above)
Comment (jadalilleboe) [reply static/js/components/GcnSelectionForm.jsx:193]: Can it just stay commented until there's some sort of request being made? the handler doesn't do anything right now.

Comment (mcoughlin) [reply static/js/components/GcnSelectionForm.jsx:193]: Ah of course!

File: static/js/components/ObservationPlanRequestLists.jsx (hunk 3 above)
Comment (jadalilleboe) [reply static/js/components/ObservationPlanRequestLists.jsx:112]: yes, default for all data points to be unselected.

**Pull request merged**